<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gir::Internal::GirWidget</class>
 <widget class="QWidget" name="Gir::Internal::GirWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_16">
     <item>
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>Score Gir</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="girLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>DÃ©pendant</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="girBar">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>6</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>Autonome</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTreeView" name="treeView">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="GR_2"/>
  <buttongroup name="GR_12"/>
  <buttongroup name="GR_3"/>
  <buttongroup name="GR_4"/>
  <buttongroup name="GR_5"/>
  <buttongroup name="GR_6"/>
  <buttongroup name="GR_7"/>
  <buttongroup name="GR_8"/>
  <buttongroup name="GR_9"/>
  <buttongroup name="GR_0"/>
  <buttongroup name="GR_10"/>
  <buttongroup name="GR_1"/>
  <buttongroup name="GR_11"/>
 </buttongroups>
</ui>
